frontend/packages/console-app/src/components/nodes/node-dashboard/queries.ts

```diff
@@ -35,10 +35,8 @@ const queries = {
     `node_memory_MemTotal_bytes{instance='<%= node %>'} - node_memory_MemAvailable_bytes{instance='<%= node %>'}`,
   ),
   [NodeQueries.MEMORY_TOTAL]: _.template(`node_memory_MemTotal_bytes{instance='<%= node %>'}`),
-  [NodeQueries.POD_COUNT]: _.template(`kubelet_running_pod_count{instance=~'<%= ipAddress %>:.*'}`),
-  [NodeQueries.FILESYSTEM_USAGE]: _.template(
-    `instance:node_filesystem_usage:sum{instance='<%= node %>'}`,
-  ),
+  [NodeQueries.POD_COUNT]: _.template(`kubelet_running_pods{instance=~'<%= ipAddress %>:.*'}`),
+  [NodeQueries.FILESYSTEM_USAGE]: _.template(`node_filesystem_avail_bytes{instance="<%= node %>"}`),
```

Comment:
@rebeccaalpert This tracks available bytes, not what's used, right? This looks incorrect to me. We'll wait for the monitoring team to weigh in.

Reply:
They suggested sum (node_filesystem_avail_bytes{instance="<%= node %>"}), but it was returning negative values.

Reply:
`sum(node_filesystem_size_bytes{instance="<%= node %>",fstype!=""} - node_filesystem_avail_bytes{instance="<%= node %>",fstype!=""})` should do it.

@sur for a second look.

Reply:
The numbers aren't the same, but they're closer with this.

<img width="1005" alt="Screen Shot 2020-08-25 at 10 00 09 AM" src="https://user-images.githubusercontent.com/7014965/91183749-d54aaa80-e6b9-11ea-86aa-f9a2886b554e.png">

<img width="469" alt="Screen Shot 2020-08-25 at 10 00 19 AM" src="https://user-images.githubusercontent.com/7014965/91183767-db408b80-e6b9-11ea-8b51-893f735b3fe9.png">

Reply:
The `486.1GiB` value don't feel correct here...

<img width="1005" alt="91183749-d54aaa80-e6b9-11ea-86aa-f9a2886b554e" src="https://user-images.githubusercontent.com/2587585/91184987-8dde1180-e6ed-11ea-809c-72a46e159ed8.png">

Reply:
I suspect the value is summed by node's role (e.g. worker/master).

![image](https://user-images.githubusercontent.com/2587585/91185160-c120a080-e6ed-11ea-865c-6ea61f17b0fa.png)

Reply:
the query expressed in https://github.com/openshift/console/pull/6419#discussion_r476443738 lgtm :+1: